boat-quay/boat-quay-rules/src/main/kotlin/com/backbase/oss/boat/quay/ruleset/InfoBlockTitleChecker.kt

```diff
@@ -0,0 +1,24 @@
+package com.backbase.oss.boat.quay.ruleset
+
+import com.typesafe.config.Config
+import org.zalando.zally.core.toJsonPointer
+import org.zalando.zally.rule.api.*
+
+@Rule(
+        ruleSet = BoatRuleSet::class,
+        id = "B005",
+        severity = Severity.MUST,
+        title = "Check info block title format."
+)
+class InfoBlockTitleChecker {
+
+    private val MAX_TITLE_LENGTH = 35
```

Comment:
Can't you make this configurable from the .conf?

Reply:
done!